--- pm-utils-1.2.5/pm/sleep.d/98smart-kernel-video.old	2009-04-03 20:48:21.000000000 +0200
+++ pm-utils-1.2.5/pm/sleep.d/98smart-kernel-video	2010-05-26 15:25:32.366717703 +0200
@@ -34,7 +34,7 @@
     # attempt to run any quirks
     [ -d /sys/class/drm/card0/device/graphics/fb0 ] || return 1
     remove_all_video_quirks
-    add_parameter --quirk-no-chvt
+    add_parameters --quirk-no-chvt
 }
 
 have_nvidia()
